src/widgets/tabview/lv_tabview.c

```diff
@@ -107,6 +107,30 @@ void lv_tabview_rename_tab(lv_obj_t * obj, uint32_t idx, const char * new_name)
     lv_label_set_text(label, new_name);
 }
 
+#if LV_USE_TRANSLATION
+
+lv_obj_t * lv_tabview_add_translation_tag_tab(lv_obj_t * obj, const char * tag)
```

Comment:
This function name seems a little weird. I'd keep only `lv_tabview_set_tab_translation_tag`, so the user can create a tab (maybe with an empty string) and set a translated text.